src/components/popover-container/popover-container-test.stories.tsx

```diff
@@ -97,3 +98,43 @@ export const PopoverContainerWithSelect = () => {
     </div>
   );
 };
+
+export const InAScrollableBlock = () => {
+  return (
+    <Box>
+      <Box bg="#ccd6dbff" height={500} width={1100} />
+      <Box height={400} overflow="scroll">
+        <Box height={400} position="fixed">
+          <PopoverContainer
+            title="This is the title"
+            disableAnimation
+            renderOpenComponent={({
+              "data-element": dataElement,
+              onClick,
+              ref,
+              "aria-label": ariaLabel,
+              id,
+              "aria-expanded": ariaExpanded,
+              "aria-haspopup": ariaHasPopup,
+            }) => (
+              <Button
+                iconType="settings"
+                iconPosition="after"
+                data-element={dataElement}
+                aria-label={ariaLabel}
+                aria-haspopup={ariaHasPopup}
+                aria-expanded={ariaExpanded}
+                forwardRef={ref}
```

Comment:
```suggestion
                ref={ref}
```

forwardRef is marked for deprecation